score_normalization_v2_for_toxicity.md

```diff
@@ -0,0 +1,174 @@
+
+We plan to start returning the updated Toxicity scores on **Tuesday,
+2017-08-15**. We will send a message to
+[perspective-announce](http://groups.google.com/forum/#!forum/perspective-announce)
+once the rollout is complete.
+
+
+## What should I do?
+
+To avoid using out-of-date score thresholds for Toxicity in your application,
+you can:
+* Consult the [toxicity score mappings](#score-mappings) below to update your
+  thresholds.
+* Configure your application for a
+  [controlled migration](#i-want-to-control-when-i-migrate).
+
+
+## What's changing and why?
+
+Our normalization process applies a version of [probability
+calibration](http://scikit-learn.org/stable/modules/calibration.html) to make
+the scores approximate probabilities. For example, so that a toxicity score of
+**0.80** can be interpreted as a belief that **80%** of people would consider
+the comment to be toxic.
+
+And yeah, sorry we're doing this again. Our first attempt at doing this wasn't
+quite right for the case when we have many annotations per example. Given we
+have the distribution of crowd-worker ratings for the Toxicity model, we
+realised we should be calibrating directly on those (we were initially,
+accidentally, calibrating on the judgement of > 50% of participants rating the
+example toxic - subtle, but a little different!). For other models based on the
+NYT annotations, our callibration was already correct because there was only a
+single decision, not a distribution of them.
+
+
+## I want to control when I migrate
+
+You can control when you migrate to normalized scores by requesting **versioned
+model attribute names**. Using this method, you can atomically update your
+application to use updated thresholds and receive normalized scores.
+
+A controlled migration process would be:
+1.  Configure your application to use a versioned attribute name before the
+    rollout.
+    * You can choose, either `TOXICITY@2` for uncalibrated or `TOXICITY@3` for
+      our first attempt at calibration, which is currently serving if you
+      request just `TOXICITY`).
+2.  Sometime the rollout - whenever you feel like it - re-configure your
+    application to use unversioned, normalized attribute names along with
+    updating your score thresholds, according to the [score
+    mappings](#score-mappings).
+
+Notes:
+
+* When you request a versioned attribute, the attribute names in the [response
+  object](https://github.com/conversationai/perspectiveapi/blob/master/api_reference.md#analyzecomment-response)
+  will also include the version (e.g., if you request `TOXICITY@3` you'll get
+  scores for `TOXICITY@3` instead of just `TOXICITY`). This allows you request
+  multiple versions of the same attribute. However, most users will probably
+  want to remove this version specification from the attribute name (for
+  example, by using a regex to remove `@.*` from the attribute name).
+* If you want to migrate to fixed normalized scores now, you can do so by using
```

Comment:
Maybe move this to be the first point so that all the versions are mentioned closer together. If someone skims over the previous point, they may miss this one.